src/main/java/uk/gov/laa/ccms/caab/controller/application/summary/EditProceedingsAndCostsSectionController.java

```diff
@@ -1271,4 +1286,292 @@ public String caseCostsPost(
     return "redirect:/application/proceedings-and-costs#costs";
   }
 
+  /**
+   * Displays the prior authority type selection page.
+   *
+   * @param model the model to add attributes to for the view.
+   * @return the view name for prior authority type selection.
+   */
+  @GetMapping("/application/prior-authorities/add/type")
```

Comment:
It looks a bit wrong having the action type in the middle of the path. Should it not be at the end?